plugins/cilium-cni/types/types_test.go

```diff
@@ -132,8 +130,7 @@ func (t *CNITypesSuite) TestReadCNIConfENI(c *check.C) {
   "type": "cilium-cni",
   "eni": {
     "instance-type": "m4.xlarge",
-    "pre-allocate": 16,
-    "first-interface-index": 2,
+	"first-interface-index": 2,
```

Comment:
It seems there is an indentation change here, this line should not be modified.

Reply:
this one looks okay in my IDE. Not sure why here looks like the indentation problem